### ISSUE TITLE
Suggest to use Forum for new features

### ISSUE THREAD

**Issue by benjaoming** [django/code.djangoproject.com#249]
Background in a recent discussion: https://fosstodon.org/@alextomkins/113674893282747412

The page https://code.djangoproject.com/newticket has the following instruction:

> Report bugs and suggest features in Django itself (or its documentation) using this ticket tracker.

The text could perhaps be changed to:

> Report bugs and suggest documentation changes using this ticket tracker.
>
> New features are recommended to be discussed firstly on [The Django Forum](https://forum.djangoproject.com/) - once feedback is gathered, use the ticket tracker to open a ticket for the new feature. Please link existing discussions from Forum (or elsewhere).

Optional addition:

> If your feature is a smaller improvement, please feel free to skip the initial feedback gathering.

**Comment by carltongibson** [comment]
Yes, this is a good idea. It's just not correct to suggest Trac tickets for new features when these will regularly be bounced to the Forum.

**Comment by benjaoming** [comment]
I've moved the latter part "If your feature is a smaller improvement, please feel free to skip the initial feedback gathering" as an optional addition since I'm not sure if that unnecessarily invites people to want to perceive their feature as simple/straight-forward without asking anyone's opinion. Maybe even for a simple feature, it's good to have a second opinion.
